packages/server-wallet/src/db/migrations/20201211180815_ajudicator_status.ts

```diff
@@ -1,11 +1,13 @@
 import * as Knex from 'knex';
 
-const tableName = 'challenge_status';
+const tableName = 'adjudicator_status';
 export async function up(knex: Knex): Promise<void> {
   await knex.schema.createTable(tableName, function (table) {
     table.string('channel_id').notNullable();
     table.integer('finalizes_at').notNullable();
     table.integer('block_number').notNullable();
+    table.integer('block_timestamp').notNullable();
+    table.specificType('states', 'jsonb[]');
```

Comment:
Does this need to be an array of `jsonb`? I _think_ `jsonb` allows for arrays.
```suggestion
    table.specificType('states', 'jsonb');
```

Reply:
I ran into [this issue](https://jawj.github.io/zapatos/#casting-parameters-to-json) where postgres gets confused by the array and won't automatically `JSON.stringify/parse` for us. By using `jsonb[]` we can let postgres handle the stringifying and parsing.